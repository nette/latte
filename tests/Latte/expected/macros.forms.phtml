<?php
%A%
Nette\Latte\Macros\FormMacros::renderFormBegin($form = $_form = $_control["myForm"], array('id' => 'myForm', 'class'=>"ajax")) ;$iterations = 0; foreach ($form->controls as $name => $field): ?>
		<?php $_input = is_object($name) ? $name : $_form[$name]; if ($_label = $_input->getLabel()) echo $_label->addAttributes(array())  ?>

<?php $_input = is_object($name) ? $name : $_form[$name]; echo $_input->getControl()->addAttributes(array('title' => 'Hello', 'cols' => 10)) ?>

		<br />

		<?php $_input = is_object($field) ? $field : $_form[$field]; if ($_label = $_input->getLabel()) echo $_label->addAttributes(array('title' => 'hello'))->startTag() ?>
 <?php $_input = is_object($field) ? $field : $_form[$field]; echo $_input->getControl()->addAttributes(array('title' => 'Hello', 'cols' => 10)) ?>
 </label>
<?php $iterations++; endforeach ?>

	<?php $_input = is_object($form['username']) ? $form['username'] : $_form[$form['username']]; if ($_label = $_input->getLabel()) echo $_label->addAttributes(array())  ?>


	<LABEL title=hello<?php echo $_form["username"]->getLabel()->addAttributes(array (
  'title' => NULL,
))->attributes() ?>>Name</LABEL>
	<input value=val type class="hello" <?php echo $_form["username"]->getControl()->addAttributes(array (
  'value' => NULL,
  'type' => NULL,
  'class' => NULL,
))->attributes() ?>/>

	<label<?php $_input = is_object($form['username']) ? $form['username'] : $_form[$form['username']]; echo $_input->getLabel()->attributes() ?>></label>
	<input <?php $_input = is_object($form['username']) ? $form['username'] : $_form[$form['username']]; echo $_input->getControl()->attributes() ?>/>
<?php Nette\Latte\Macros\FormMacros::renderFormEnd($_form) ?>


<?php Nette\Latte\Macros\FormMacros::renderFormBegin($form = $_form = $_control["myForm"], array()) ;Nette\Latte\Macros\FormMacros::renderFormEnd($_form) ?>



<input <?php $_input = is_object($form['username']) ? $form['username'] : $_form[$form['username']]; echo $_input->getControl()->attributes() ?>/>


<?php $iterations = 0; foreach ($form['sex']->items as $key => $label): ?>
	<?php if ($_label = $_form["sex"]->getLabel(NULL, $key)) echo $_label->addAttributes(array())->startTag() ?>
 <?php echo $_form["sex"]->getControl($key)->addAttributes(array()) ?> <?php echo Nette\Templating\Helpers::escapeHtml($label, ENT_NOQUOTES) ?>
</label>
<?php $iterations++; endforeach ;
