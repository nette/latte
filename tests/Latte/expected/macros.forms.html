<form action="" method="post"><table>
<tr>
	<th><label for="frm-username">Username:</label><th>
	<td><input title="Hello" cols="10" type="text" name="username" id="frm-username" value="" /></td>
</tr>
<tr>
	<th><label for="frm-password">Password:</label><th>
	<td><input title="Hello" cols="10" type="password" name="password" id="frm-password" /></td>
</tr>
<tr>
	<th><label>Sex:</label><th>
	<td><input type="radio" name="sex" id="frm-sex-m" value="m" /><label for="frm-sex-m">male</label><br /><input type="radio" name="sex" id="frm-sex-f" value="f" /><label for="frm-sex-f">female</label><br /></td>
</tr>
<tr>
	<th><th>
	<td><input title="Hello" cols="10" type="submit" name="send" id="frm-send" value="Sign in" /></td>
</tr>
</table>
</form>

		 <input title="Hello" cols="10" type="hidden" name="id" id="frm-id" value="" /> </label>
		<label for="frm-username">Username:</label>
<input title="Hello" cols="10" type="text" name="username" id="frm-username" value="" />
		<br />

		<label title="hello" for="frm-username"> <input title="Hello" cols="10" type="text" name="username" id="frm-username" value="" /> </label>
		<label>Sex:</label>
<label for="frm-sex-m"><input type="radio" name="sex" id="frm-sex-m" value="m" />male</label><br /><label for="frm-sex-f"><input type="radio" name="sex" id="frm-sex-f" value="f" />female</label><br />
		<br />

		<label title="hello"> <label for="frm-sex-m"><input type="radio" name="sex" id="frm-sex-m" value="m" />male</label><br /><label for="frm-sex-f"><input type="radio" name="sex" id="frm-sex-f" value="f" />female</label><br /> </label>

<input title="Hello" cols="10" type="submit" name="send" id="frm-send" value="Sign in" />
		<br />

		 <input title="Hello" cols="10" type="submit" name="send" id="frm-send" value="Sign in" /> </label>

	<label for="frm-username">Username:</label>

	<LABEL title=hello for="frm-username">Name</LABEL>
	<input value=val type class="hello"  name="username" id="frm-username"/>

	<label for="frm-username"></label>
	<input  type="text" name="username" id="frm-username" value=""/>
<div><!--[if IE]><input type=IEbug disabled style="display:none"><![endif]--></div>
</form>


<form action="" method="post"><div><!--[if IE]><input type=IEbug disabled style="display:none"><![endif]--></div>
</form>



<input  type="text" name="username" id="frm-username" value=""/>


	<label for="frm-sex-m"> <input type="radio" name="sex" id="frm-sex-m" value="m" /> male</label>
	<label for="frm-sex-f"> <input type="radio" name="sex" id="frm-sex-f" value="f" /> female</label>
